src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c

```diff
@@ -80,7 +80,7 @@ static InitChainEntry sInitChain2[] = {
     ICHAIN_F32(uncullZoneDownward, 700, ICHAIN_STOP),
 };
 
-static BgSpot18ObjInitFunc D_808B913C[] = {
+static BgSpot18ObjInitFunc sInitFuncs2[] = {
```

Comment:
then this can drop the 2
```suggestion
static BgSpot18ObjInitFunc sInitFuncs[] = {
```